### ISSUE TITLE
[sklearn#34580] RFC Should we require contributors to communicate in their own words (as opposed to posting AI output)?

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#314]
**Mirrored from [scikit-learn#34580](https://github.com/scikit-learn/scikit-learn/issues/34580)**

Original title: RFC Should we require contributors to communicate in their own words (as opposed to posting AI output)?
Original labels: RFC
Created: 2026-07-27T10:06:31Z

---

Here I propose to require contributors to write comments, PR descriptions and issues using their own words, as opposed to posting AI generated text.

Specifically, I really like the [Astral AI policy](https://github.com/astral-sh/.github/blob/main/AI_POLICY.md), that explicitly states:

> AI should not be used to generate comments when communicating with maintainers. We expect comments on our projects to be written by humans. We may hide any comments that we believe are AI generated.
> ...
> If you are opening an issue, we expect you to describe the problem in your own words.
> ...
> If you wish to include context from an interaction with AI in your comments, it must be in a quote block (e.g., using >) and disclosed as such. It must be accompanied by human commentary explaining the relevance and implications of the context. Do not share long snippets.

Using AI to post long generated summaries on issues and PR descriptions and to comment on how to resolve a certain problem is very impolite / disrespectful anyway. It puts the burden on filtering out the useful information (if any) on the reader. The poster may not be the author and may not understand what they were posting.

If we require contributors to use their own words in written communication we can easier handle the extreme cases by either requests to use their own words or by closing if genuine communication is not possible.

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Moved to https://github.com/godofecht/flow-scikit/issues/109 in the flow-scikit repository. Further library-specific triage and implementation will happen there.
